configure.ac

```diff
@@ -330,6 +330,10 @@ if test "$my_htop_platform" = darwin; then
    AC_CHECK_FUNCS([mach_timebase_info])
    AC_CHECK_DECLS([IOMainPort], [], [], [[#include <IOKit/IOKitLib.h>]])
    AC_CHECK_DECLS([IOMasterPort], [], [], [[#include <IOKit/IOKitLib.h>]])
+
+   AC_CHECK_FUNCS([host_statistics64], [
```

Comment:
AC_CHECK_FUNCS does not take the fourth argument. The `#include <mach/mach.h>` is unnecessary and would be ignored by Autoconf.